src/nora/components/Tag/Tag.js

```diff
@@ -5,11 +5,17 @@ import styles from './Tag.module.scss'
 export const Tag = ({ children, type }) => {
   let colorClass
   switch (type) {
-    case 'approved':
```

Comment:
The `propTypes` were not updated to reflect this so we get console warnings as approved and rejected are currently only valid inputs.

Easy peasy fix and I’m just starting my day so I’ll update Tag.js 🙂

I may add some test coverage too

Reply:
Got it! Thanks Rob :)